src/python/pants/backend/nfpm/rules.py

```diff
@@ -157,12 +160,13 @@ async def package_nfpm_rpm_package(field_set: NfpmRpmPackageFieldSet) -> BuiltPa
     return built_package
 
 
-def rules():
+def rules() -> Iterable[Rule | UnionRule]:
```

Comment:
This added typehint is not strictly required by this PR. It's a minor consistency improvement (+4 -2 lines), and I'm already editing this file (+2 lines), so I included it here.